runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/blog/BreadcrumbsSection/BreadcrumbsSection.module.scss

```diff
@@ -1,5 +1,49 @@
+@import '#src-landing/scss/main.scss';
+
+$SECTION-HEIGHT: 59px;
```

Comment:
why 59?

Reply:
do we really need to specify height here? imo just add padding around it to make sure other elements wont overlap it. It should scale better